Library/Homebrew/requirements/apr_requirement.rb

```diff
@@ -4,7 +4,7 @@ class AprRequirement < Requirement
   fatal true
   default_formula "apr-util"
 
-  satisfy(:build_env => false) { MacOS::CLT.installed? }
+  satisfy(:build_env => false) { MacOS.version > :leopard && MacOS::CLT.installed? }
```

Comment:
A comment to explain the `:leopard` would be handy here; presumably it's just not provided these.